services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker.py

```diff
@@ -1,48 +1,62 @@
+import functools
 import logging
 from typing import Awaitable, Callable
 
 from fastapi import FastAPI
-from servicelib.background_task import start_periodic_task, stop_periodic_task
-from servicelib.redis_utils import exclusive
-from settings_library.prometheus import PrometheusSettings
+from models_library.rabbitmq_messages import _RabbitResourceTrackingBaseMessage
+from servicelib.logging_utils import log_catch, log_context
+from servicelib.rabbitmq import RabbitMQClient, RabbitSettings
 
 from .core.settings import ApplicationSettings
-from .modules.redis import get_redis_client
-from .resource_tracker_core import collect_container_resource_usage_task
-
-_TASK_NAME = "periodic_prometheus_polling"
+from .modules.rabbitmq import get_rabbitmq_client
 
 _logger = logging.getLogger(__name__)
 
 
+async def _process_message(app: FastAPI, data: bytes) -> bool:
+    # TODO: parse the message and process it
+
+    _logger.info("Received message")
+    return True
+
+
+async def _subscribe_to_rabbitmq(app) -> None:
+    with log_context(_logger, logging.INFO, msg="Subscribing to rabbitmq channel"):
+        rabbit_client: RabbitMQClient = get_rabbitmq_client(app)
+        await rabbit_client.subscribe(
+            _RabbitResourceTrackingBaseMessage.get_channel_name(),
```

Comment:
change that private class to become public. is pylint not complaining?

Reply:
done :+1: